src/screens/SettingsScreen.js

```diff
@@ -194,6 +195,17 @@ export const SettingsScreen = () => {
       setSectionedData(additionalSectionedData);
     };
 
+    if (isSupported) {
+      setFilter([
+        ...filter,
+        {
+          id: TOP_FILTER.AR_DOWNLOAD_LIST,
+          title: texts.settingsTitles.tabs.arSettings,
+          selected: false
+        }
+      ]);
+    }
+
```

Comment:
i suggest an own effect for this part as it has nothing to do with the sectioned data. additionally the effect for this should depend on the variable checked to reduce rerenderings.

does it need to be set per effect at all? couldn't the initial filter directly depend on the info about supported devices?

Reply:
I changed `INITIAL_FILTER` a bit. I made it a function that returns an array. I changed the filters in the array according to `isARSupported`.

https://github.com/ikuseiGmbH/smart-village-app-app/pull/593/commits/5b917e75ba973aa000fa55701615b059d8114296